app/templates/admin/createEvent.html

```diff
@@ -42,6 +42,82 @@
+              <input autocomplete="off" type='text' class="form-control datePicker startDatePicker readonly" 
+                value="{{ eventData.startDate.strftime('%m-%d-%Y') if eventData.startDate and eventData.startDate.strftime else eventData.startDate}}" 
+                name="startDate" placeholder="Pick a start date" id='startDatePicker-{{pageLocation}}' data-page-location="{{pageLocation}}" required>
+              <div class="input-group-text" id="calendarIconStart-{{pageLocation}}">
+                  <span><i class="bi bi-calendar-plus-fill"></i></span>
+              </div>
+          </div>
+      </div>
+
+      <div class="form-group col-md-6 {{hideDate}} endDateStyle" id ="endDateStyle-{{pageLocation}}" >
+      <!--datePicker for End date-->
+              <span><i class="bi bi-calendar-plus-fill"></i></span>
+          <label class="form-label me-2" for="endDate-{{pageLocation}}" ><strong>End Date</strong></label>
+          <div class='input-group date endDate' id="endDate-{{pageLocation}}" data-page-location="{{pageLocation}}">
+            <input autocomplete="off" type='text' class="form-control datePicker endDatePicker" id='endDatePicker-{{pageLocation}}' 
+              value="{{eventData.endDate.strftime('%m-%d-%Y') if eventData.endDate and eventData.endDate.strftime else eventData.endDate}}" 
+              name="endDate" placeholder="Pick an end date" data-page-location="{{pageLocation}}">
+            <div class="input-group-text" id="calendarIconEnd-{{pageLocation}}">
+              <span><i class="bi bi-calendar-plus-fill"></i></span>
+            </div>
+          </div>
+      </div>
+    
+    </div>
+    <br>
+    <!-- Start and End Time -->
+    <div class="mb-4">
+      <div class="row col-md-12" >
+        <div class="form-group col-md-6">
+          <label class="form-label me-2" for="startTime-{{pageLocation}}"><strong>Start Time</strong></label>
+          <div class="input-group">
+            {% if eventData.timeStart %}
+              {% set start_time = eventData.timeStart %}
+            {% else %}
+              {% set start_time = "12:00" %}
+            {% endif %}
+            <input autocomplete="off" type="time" class="form-control timepicker" value="{{start_time}}" name="timeStart" 
+              aria-describedby="timeIcon1" placeholder="Pick a start time"  
+              id="startTime-{{pageLocation}}" data-page-location="{{pageLocation}}" required />
+            <span class="input-group-text timeIcons" id="timeIcon1-{{pageLocation}}" hidden><i class="bi bi-clock"></i></span>
+          </div>
+        </div>
+        <div class='form-group col-md-6'>
+          <label class="form-label me-2" for="endTime-{{pageLocation}}"><strong>End Time</strong></label>
+          <div class="input-group">
+            {% if eventData.timeEnd %}
+              {% set end_time = eventData.timeEnd %}
```

Comment:
Other variables in this file are camel case.

Reply:
Fixed.